org.eclipse.scout.rt.jackson/src/main/java/org/eclipse/scout/rt/jackson/dataobject/DoEntityDeserializer.java

```diff
@@ -214,16 +216,26 @@ protected IDoEntity resolveEntityType(DeserializationContext ctxt, String entity
       // try to lookup DoEntity with specified entityType
       Class<? extends IDoEntity> clazz = m_doEntityDeserializerTypeStrategy.resolveTypeName(entityType);
       if (clazz != null) {
+        // (1) Class could be resolved by given entityType, validate that class is assignable from class handled (e.g. expected to be created) by this deserializer instance
```

Comment:
Something in the comment got mixed-up with the ()

Reply:
Clarified comment